spec/daru/io/importers/csv_spec.rb

```diff
@@ -1,4 +1,9 @@
-RSpec.describe Daru::IO::Importers::CSV do # rubocop:disable Metrics/BlockLength
+RSpec.shared_examples 'csv importer' do
```

Comment:
Nothing really, but let this (issue #24) and passing readable spec like 
```Ruby
    it_behaves_like 'exact dataframe',
        order: ...,
        index: ...,
        data : ...,
        nrows: ...,
        ncols: ...
```
 be left for a separate PR. Probably in PR #27 😄